README.md

```diff
@@ -286,6 +286,19 @@ Java
     spConsentLib.loadPrivacyManager("<PM_ID>", CampaignType.CCPA); // For a CCPA campaign
 ```
 
+Another option to use to **resurface** the privacy manager, is the following:
```

Comment:
@andresilveirah

Reply:
@carmelo-iriti if the method `loadPrivacyManager` is generic (ie. it works for all types of PM) why are we limiting its 3rd parameter to an OTT type?

What if I want to load a "app" Privacy manager?

Reply:
we need to add in the `OttPmType` an APP item or we can delete the `generic ` word in the description